includes/discovery/sensors/pre-cache/esphome.inc.php

```diff
@@ -0,0 +1,65 @@
+<?php
```

Comment:
This file is not needed im pretty sure

Reply:
Check out this discussion @laf and I had on the LibreNMS Community https://community.librenms.org/t/new-os-data-shows-in-discovery-but-not-in-poller/26180

Data does not populate without it.